homeassistant/components/tuya/config_flow.py

```diff
@@ -25,7 +25,9 @@
 
 RESULT_SINGLE_INSTANCE = "single_instance_allowed"
 RESULT_AUTH_FAILED = "invalid_auth"
-TUYA_ENDPOINT_BASE = "https://openapi.tuyacn.com"
+TUYA_ENDPOINT_BASE_CHINA = "https://openapi.tuyacn.com"
+TUYA_ENDPOINT_BASE_US = "https://openapi.tuyaus.com"
+COUNTRY_CODE_CHINA = ["86", "+86", "China"]
```

Comment:
Is there a list of all valid country codes somewhere?

Reply:
The documentation points to <https://www.countrycode.org/>, which indicates that this should be `COUNTRY_CODE_CHINA = "86"`